NEWS.md

```diff
@@ -14,6 +14,8 @@
 
 ## Breaking changes
 
+* Signature for `toUnit` function has changed. `molWeight` is now the fifth, instead of fourth, parameter (#837).
```

Comment:
maybe you could say that sourceUnit is the fourth parameter as well :)

Reply:
We coddle our users so much 🤪

Done!